simple_graph/weighted_graph.py

```diff
@@ -0,0 +1,170 @@
+#! /usr/bin/env python
+
+# Simple Graph project for Efrain, Henry, Mark
+# Graph is unweighted but directed
+
+import time
+from collections import OrderedDict
+
+
+def timed_func(func):
+    """Decorator for timing our traversal methods."""
+    def timed(*args, **kwargs):
+        start = time.time()
+        result = func(*args, **kwargs)
+        elapsed = time.time() - start
+        print "time expired: %s" % elapsed
+        return result
+    return timed
+
+
+class Wgraph(object):
+    """A class defining a unweighted direct simple graph."""
+
+    def __init__(self):
+        """Instantiates simple graph"""
+        self.dict = {}
```

Comment:
since the edge container is an OrderedDict, is there a reason to use one here at the top level too?